src/WaCore.Crud.Contracts/Web/IWacListDataVm.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using WaCore.Crud.Contracts.Dtos;
+
+namespace WaCore.Crud.Contracts.Web
+{
+    public interface IWacListDataVm<TDto, TFilter>
+        where TDto : class
+        where TFilter : IWacFilter
+    {
+        List<TDto> DtoList { get; set; }
```

Comment:
Here I think we also should replace List class with some interface.

Reply:
I'm already changed this (including other changes), but I'm going to make a separate PR for that.